nanostack-event-loop/eventOS_event.h

```diff
@@ -19,6 +19,173 @@
 extern "C" {
 #endif
 
+/**
+ * \defgroup nanostack-eventloop Nanostack's event system.
+ * Small event scheduler and timer system written in C.
+ *
+ * This event system is originating from project called Nanostack and developed within Arm. Therefore
+ * Some of the types and names within this library are prefixed with `ns_*` or `arm_*` or `eventOS*`.
+ *
+ * <h3>Concept</h3>
+ *
+ * Event loop uses a concept called tasklet, which is just a callback function that receives events.
+ * There can be as many as 254 tasklets registered if memory allows. This is only limited by event ID being just 8-bits.
```

Comment:
The tasklet id is signed 8bit int and negative return value from eventOS_event_handler_create is a error so the max amount of tasklets is 128.

Reply:
Ah.. true..